lib/crawler/keyword/keywords.ex

```diff
@@ -24,6 +24,9 @@ defmodule Crawler.Keyword.Keywords do
   def get_keyword_by_user_id_and_id!(user_id, id),
     do: Repo.get_by!(Keyword, id: id, user_id: user_id)
 
+  def get_keyword_by_user_id_and_id(user_id, id),
+    do: Repo.get_by(Keyword, id: id, user_id: user_id)
```

Comment:
Just FYI, we can use the `get_by!` and let it raise the exception like the Web version, and we could handle the `not_found` template for JSON format. That requires some additional setup in the view, it's out of the IC's scope ==> So I'm ok with the current implementation 👌